src/SEOTools/SEOMeta.php

```diff
@@ -181,6 +197,22 @@ public function generate($minify = false)
             $html[] = "<meta {$name}=\"{$key}\" content=\"{$content}\">";
         }
 
+        foreach ($preloads as $preload) {
+            $href = $preload[0];
+            $as = $preload[1];
+
+            // if $href is empty jump to next
+            if (empty($href)) {
+                continue;
+            }
```

Comment:
Unnecessary empty check: since the `addPreload` function already requires both `href` & `as` to be present.

Reply:
yes, you are right. removing it.